BrewMesModulesParent/common/src/main/java/com/brewmes/common/services/IScheduleService.java

```diff
@@ -0,0 +1,46 @@
+package com.brewmes.common.services;
+
+import com.brewmes.common.entities.ScheduledBatch;
+
+import java.util.List;
+
+public interface IScheduleService {
+
+    /**
+     * Adds a batch to the global queue.
+     *
+     * @param speed    Speed of the batch.
+     * @param beerType Beer type of the batch.
+     * @param amount   Amount to produce.
+     * @return the placement of the added {@code ScheduledBatch} in the queue.
+     */
+    int addToQueue(int speed, int beerType, int amount);
```

Comment:
```suggestion
    int addToQueue(ScheduledBatch scheduledBatch);
```

Reply:
This is because we can get ScheduledBatch objects directly form the @RequestBody in the controller

Reply:
What does this method return if it fails? I think we should add it to the JavaDoc?